src/components/Exchange.js

```diff
@@ -1087,29 +1093,26 @@ export default class Exchange extends React.Component {
                  this.setState({xdaiToDendaiMode:"withdraw"})
                }}>
                  <Scaler config={{startZoomAt:400,origin:"50% 50%"}}>
-                  <i className="fas fa-arrow-down" /> {this.props.ERC20NAME} to xDai
+                  <i className="fas fa-arrow-down" /> {token.name} to xDai
                  </Scaler>
                </button>
              </div>
            </div>
         )
       }
 
-      let link = ""
-      if(this.props.contracts){
-        link = "https://blockscout.com/poa/dai/address/"+this.props.contracts[this.props.ERC20TOKEN]._address+"/contracts"
-      }
+      const link = `https://blockscout.com/poa/dai/address/${token.address}/contracts`;
```

Comment:
More LeapDAO/burner-wallet config.js propaganda: https://github.com/leapdao/burner-wallet/blob/master/src/components/Receive.js#L45 :)

Reply:
This part does seem like it would make sense in a config file, since it's only client-facing. However, it's probably out of scope for this PR :)

Reply:
Yes it is out of scope. Just wanted to promote config.js :D